src/pages/Home/index.jsx

```diff
@@ -10,11 +11,18 @@ const Home = () => {
 
     const formData = new FormData(event.target);
     const repositoryURL = Object.fromEntries(formData.entries()).repositoryURL;
-    const { organizationName, repositoryName } =
-      extractGitInfoFromURL(repositoryURL);
+    const { owner, repo } = extractGitInfoFromURL(repositoryURL);
 
     try {
-      getCommitList({ organizationName, repositoryName, setCommitList });
+      const commitList = await getCommitList({ owner, repo });
+      const checkCommitList = getCheckCommitList(commitList);
+      const changeList = await getCommitDiffList({
+        owner,
+        repo,
+        checkCommitList,
+      });
+
+      setCommitList(changeList);
```

Comment:
```suggestion
const changeList = await getCommitDiffList({
        owner,
        repo,
        checkCommitList,
      });
      
const jsonMap = changeList.map((change) => Array.from(change.values()));

setCommitList(jsonMap);
```
현재 checkCommitList 가 Map 이 들어있는 배열로 반환이 되네요!
Zustand의 persist가 직렬화할 수 없는 데이터를 관리하지 못해서 Map 객체를 스토어에 저장할 수 없을 것 같아요! 그래서 위와 같이 Map을 저장할 때 JSON 직렬화 가능한 형태로 변환해서 저장하도록 우선 해주세요!

Reply:
네 반영했습니다!